desktop/cmp/error/ErrorMessage.js

```diff
@@ -18,31 +18,52 @@ import './ErrorMessage.scss';
  */
 export const [ErrorMessage, errorMessage] = hoistCmp.withFactory({
     className: 'xh-error-message',
-    render({className, error, title, actionFn, actionButtonProps}, ref) {
+    render({className, error, message, title, actionFn, actionButtonProps}, ref) {
+        if (!error) return null;
```

Comment:
If `error` is '' (empty string)  this will return null.
This behaviour conflicts with the `error` proptype doc.

Same note for the mobile cmp.